### PR TITLE
refactor: Use schema mapper for biorad bioplex

### PR DESCRIPTION
**Pull request by nathan-stender**

### CODE REVIEW

File: tests/parsers/biorad_bioplex_manager/testdata/bio-rad_bio-plex_manager_example_01.xml [hunk 1]
```diff
@@ -110012,23 +110012,23 @@
                             </MWAnalyte>
                             <MWAnalyte RegionNumber="15">
                                 <AnalyteName>bravo</AnalyteName>
-                                <Reading Valid="true" Code="0" OutlierType="None">3.60000000000000000E+001</Reading>
+                                <Reading Valid="true" Code="1" OutlierType="None">3.60000000000000000E+001</Reading>
```
Comment (nathan-stender) [inline tests/parsers/biorad_bioplex_manager/testdata/bio-rad_bio-plex_manager_example_01.xml:110015]: Note to reviewer: adding error codes to some samples to exercise error code

File: tests/parsers/biorad_bioplex_manager/testdata/bio-rad_bio-plex_manager_example_01.json [hunk 2]
```diff
@@ -256,9 +256,6 @@
                                         }
                                     }
                                 ]
-                            },
```
Comment (nathan-stender) [inline tests/parsers/biorad_bioplex_manager/testdata/bio-rad_bio-plex_manager_example_01.json:259]: Note to reviewer: this was a harmless bug in the existing implementation, should not include empty list

File: tests/parsers/biorad_bioplex_manager/to_allotrope_test.py [hunk 3]
```diff
@@ -17,15 +9,3 @@
 
 class TestParser(ParserTest):
     VENDOR = VENDOR_TYPE
-
-
-@pytest.mark.short
```
Comment (nathan-stender) [inline tests/parsers/biorad_bioplex_manager/to_allotrope_test.py:22]: Note to reviewer: moved to reader test

File: tests/parsers/biorad_bioplex_manager/testdata/bio-rad_bio-plex_manager_example_01.json [hunk 4]
```diff
@@ -3169,7 +3136,28 @@
                                 ]
                             },
                             "error aggregate document": {
-                                "error document": []
+                                "error document": [
```
Comment (nathan-stender) [inline tests/parsers/biorad_bioplex_manager/testdata/bio-rad_bio-plex_manager_example_01.json:3139]: Note to reviewer: these are the errors added to the test case

**Review by stephenworlow — DISMISSED** [review]

**Review by stephenworlow — DISMISSED** [review]

**Review by stephenworlow — APPROVED** [review]

**Pull request merged**